examples/generate-and-verify-signature/README.md

```diff
@@ -0,0 +1,31 @@
+# Generating and verify a signature
+
+This examples shows you how to generate a signature on the Blockchain Security 2Go card and how to verify a signature using the blocksec2go library.
+
+The contents of `get_reader()` and `activate_card(reader)` have already been covered in the previous example [get-card-info](../get-card-info). The content of `get_public_key(key_id)` has already been covered in the previous example [get_key_info](../get-key-info). Please reference those examples if something is unclear in these functions.
+
+To use the `generate_signature(reader, key_id, hash)` cpmmand we first need a hashed message that should be signed using the card. In the example we used "Hello World!" as our message and hashed it using the SHA256 algorithm:
+
+    hash_object = hashlib.sha256(b'Hello World!')
+    hash = hash_object.digest()
+
+It is important that we leave the hashed message in bytes since this is the format the card accepts.
+
+Next we need a keypair which will be used to sign the hashed message. Please only choose existing keypairs since otherwise there is no private key for the signing process.
```

Comment:
... existing keypairs with a signature counter higher than zero...

Reply:
This feature has now been added in PR #18